src/database/migrations/1739918380436_member_user_public_key.ts

```diff
@@ -0,0 +1,19 @@
+import { type Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+    await db.schema
+        .createTable('member_user_public_key')
+        .addColumn('id', 'uuid', (col) => col.defaultTo(sql`v7uuid()`).primaryKey())
+        .addColumn('user_id', 'uuid', (col) => col.notNull())
+        .addColumn('value', 'text', (col) => col.notNull())
+        .addColumn('fingerprint', 'text', (col) => col.notNull())
+        .addColumn('created_at', 'timestamp', (col) => col.defaultTo(sql`now()`).notNull())
+        .addColumn('updated_at', 'timestamp', (col) => col.defaultTo(sql`now()`).notNull())
```

Comment:
I added these audit columns for consistency, but I'm curious how the `updated_at` column actually works in this codebase. Is that some Kysely magic or a trigger we configure?